Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/CoronaTestRepositoryExtensions.kt

```diff
@@ -0,0 +1,26 @@
+package de.rki.coronawarnapp.coronatest
+
+import de.rki.coronawarnapp.coronatest.type.CoronaTest
+import de.rki.coronawarnapp.coronatest.type.pcr.PCRCoronaTest
+import de.rki.coronawarnapp.coronatest.type.rapidantigen.RapidAntigenCoronaTest
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.map
+
+val CoronaTestRepository.latestPCRT: Flow<PCRCoronaTest?>
```

Comment:
Since this our implementation, and it is part of the repository core functionality I prefer  to move this to the repo as regular properties not an extensions what do you think?

Reply:
I'd prefer to keep the allowed I/O operations on the repo minimal. This is just a helper method to filter the tests for a specific one. Adding specific flows for each test type to the repo feels like API bloat.